docs/en/cloud/manage/service-types.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: Service Types
+slug: /en/cloud/manage/service-types
+title: Service Types
+---
+
+# Service Types
+
+There are several service types available in ClickHouse Cloud. This page discusses which service type is right for your specific use case. It is possible to have different services in your ClickHouse Cloud organization, each of which has its own specified service type.
+
+**Summary of service types:**
+
+|  | Development | Production | Dedicated |
+|:---------|:-----|:---------|:---------|
+|**Use case**|Small workloads, prototypes|Medium-sized workloads, customer-facing applications|Applications with strict latency and isolation requirements that need extensive customization|
+|**Storage**|Up to 1TB|Unlimited|Unlimited|
+|**Memory**|16 GiB|24GiB+|Unlimited|
+|**Compute**|Burstable CPU|Dedicated CPU|Custom compute options|
+|**Backups**|Every 24h, retained for 1 day|Every 24h, retained for 2 days|Custom backup retention|
+|**Upgrades**|Automatic|Automatic|Schedulable|
+|**SLA and support**|24-hour response time|1-hour response time|Custom SLAs, assigned lead support engineer|
+
+## Development
+
+`Development` services are designed for smaller workloads and starter projects. They are the lowest-cost option in ClickHouse Cloud. Though at a lower price than our other service types, `Development` services are still designed for high reliability and are replicated across two availability zones.
+
+`Development` services do not support autoscaling and are “auto on” by default. `Development` services are best for internal projects and prototypes, and for developers just trying out ClickHouse.
```

Comment:
> and are “auto on” by default.

@justindeguzman what does this mean? idling is on by default both in Development and Prod

Reply:
Sounds good, I'll remove `and are “auto on” by default`